images/calico/tests/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# monopod:tag:k8s
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+curl -o calico.yaml https://raw.githubusercontent.com/projectcalico/calico/v3.26.1/manifests/calico.yaml
+
+sed -i.bak "s|docker.io/calico/node:.*|${NODE_IMAGE}|g" calico.yaml
+sed -i.bak "s|docker.io/calico/kube-controllers:.*|${KUBE_CONTROLLERS_IMAGE}|g" calico.yaml
+#sed -i.bak "s|docker.io/calico/cni:.*|${CNI_IMAGE}|g" calico.yaml
```

Comment:
Do we need this?

Reply:
Not yet, but that's the next image and I'd like to keep it as a reminder.